create-codespace-and-checkout.sh

```diff
@@ -10,38 +10,70 @@
 
 # set -e  # Exit on any error
 
-# Colors for output
```

Comment:
We should only use gum for nice output. It has a log option.

Missing dependencies can be printed very simple and plain.